src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -548,6 +548,7 @@ slots:
   mapping_tool:
     description: A reference to the tool or algorithm that was used to generate the
       mapping. Should be a URL pointing to more info about it, but can be free text.
+      Consider using the mapping_tool_id slot for a more standardised reference.
```

Comment:
Nitpick: I’d personally use a stronger word than “consider” here. If a semantic identifier is available for the tool, then `mapping_tool_id` SHOULD be used instead.